### PR TITLE
Fixing Sharepoint template

### PR DESCRIPTION
**Pull request by kenazk**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kenazk__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (kenazk). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by singhkay** [comment]
@kenazk Can you address the bot issues?

File: sharepoint-server-farm-ha/azuredeploy.parameters.json [hunk 1]
```diff
@@ -1,36 +1,77 @@
...
+    "value" : "new"
+  },
+  "dnsPrefix": {
+    "value" : "replaceme"
+  },
+  "spCADNSPrefix": {
+    "value" : "replaceme"
+  },
+  "storageAccountNamePrefix": {
+    "value": "replaceme"
```
Comment (singhkay) [inline sharepoint-server-farm-ha/azuredeploy.parameters.json]: Can you put "#####" here to enable automated testing?

File: sharepoint-server-farm-ha/azuredeploy.parameters.json [hunk 2]
```diff
@@ -1,36 +1,77 @@
...
+  "sqlVMSize" : {
+    "value" : "Standard_D3"
+  },
+  "witnessVMSize" : {
+    "value" : "Standard_D1"
+  },
+  "spVMSize" : {
+    "value" : "Standard_D3"
+  },
+  "domainName":{
```
Comment (singhkay) [inline sharepoint-server-farm-ha/azuredeploy.parameters.json]: "#####" here as well

File: sharepoint-server-farm-ha/vnet-with-dns-server.json [hunk 3]
```diff
@@ -39,7 +39,7 @@
       "name": "[parameters('virtualNetworkName')]",
       "type": "Microsoft.Network/virtualNetworks",
       "location": "[parameters('location')]",
-      "apiVersion": "2015-05-01-preview",
+      "apiVersion": "2014-12-01-preview",
```
Comment (singhkay) [inline sharepoint-server-farm-ha/vnet-with-dns-server.json]: Deprecated api version. Typo?

**Pull request merged**